data/lists.js

```diff
@@ -3,3 +3,168 @@
 // edit list - used by /list/:id route
 // delete list - used by /list/:id route
 // delete entry from list - used by /list/:id route
+import { users } from "../config/mongoCollections.js";
+import { ObjectId } from "mongodb";
+import * as validation from "../utils/validation.js";
+
+const exportedMethods = {
+  async create(userId, title, movies) {
+    //for validation validate if
+    //userId is valid
+    //title is a string
+    //movies is an array of integers
+    validation.checks(userId);
+    userId = userId.trim();
+    if (!ObjectId.isValid(userId)) throw "invalid object ID";
+    title = title.trim();
+    if (!movies || !Array.isArray(movies))
+      throw "You must provide an array of movies";
+    if (movies.length === 0) throw "You must supply at least one movie";
+    movies.forEach((elem) => {
+      elem = validation.checkNumber(elem, "movie");
+    });
```

Comment:
You can create a validation function in validation.js that would check if it is a movie array, then just call the function here:
```suggestion
movies = validation.checkMovieArray(movies, "Movies");
```
Look at the comments in validation.js to see how checkMovieArray should be implemented